libqtile/backend/wayland/core.py

```diff
@@ -1231,6 +1245,9 @@ def on_config_load(self, initial: bool) -> None:
             for device in [*self.keyboards, *self._pointers]:
                 device.configure(self.qtile.config.wl_input_rules)
 
+        # Apply wl_renderer option
+        self._configure_renderer()
```

Comment:
is this needed if we already do this when subscribing to startup complete?

Reply:
I was imagining that the renderer could be changed by reloading config.  Does that also fire startup complete?